package.json

```diff
@@ -33,7 +33,7 @@
   "author": "Didier Baquier",
   "license": "MIT",
   "bugs": {
-     "url": "https://github.com/cordova-sms/cordova-sms-plugin/issues"
+     "url": "https://github.com/cordova-sms/com.cordova.plugins.sms/issues"
```

Comment:
@tajinder-logiciel I don't think, changing homepage url is good idea, it will return 404.